src/Resources/config/services.yml

```diff
@@ -68,6 +68,11 @@ services:
         class: Contao\CoreBundle\Security\User\ContaoUserProvider
         public: false
 
+    contao.resources:
```

Comment:
I think for the sake of code readability the service id should be named `contao.resources_provider`.